langserver/langserver_test.go

```diff
@@ -449,7 +449,7 @@ package main; import "test/pkg"; func B() { p.A(); B() }`,
 				// "a.go:1:53": "type int int",
 			},
 			overrideGodefDefinition: map[string]string{
-				"a.go:1:40": "/goroot/src/fmt/print.go:256:6-256:13",  // hitting the real GOROOT
+				"a.go:1:71": "/goroot/src/fmt/print.go:38:6-38:11",    // hitting the real GOROOT
```

Comment:
Every small change in the src will fail this test case. Is there a better approach to this? Maybe just assert that a _valid_ file and line number is returned?

Reply:
yeah that sounds much better. It requires a bit of cleaning up in our tests, but that seems fine given this is relying on GOROOT of the user running the tests. Right now the location I have picked hasn't changed in 2 years.

Reply:
Updated to test path instead for this specific test.